roles.tf

```
@@ -2,7 +2,7 @@ resource "random_password" "role" {
   for_each = var.roles
 
   length  = 48
-  special = false
+  special = var.password_special_chars
```

Possible alternative to prevent changes when migrating resources:
```suggestion
  special = false
  
  lifecycle {
    ignore_changes = [special]
  }
```

---

I think I like the alternative more. Can always be tainted when a new password is needed after all.

---

Would probably add the other "modifiers" as well.

---

Any particular reason to add them? Other than being explicit about all of them

---

That and a little bit of "better safe than sorry". Might save us some follow-up PRs adding one or more others down the road. *Might* slightly ease the migration routes. If it's worth it, no idea :wink: